src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3788,7 +3788,7 @@
     <value>Scroll to previous folder when navigating up</value>
   </data>
   <data name="NewWindowDescription" xml:space="preserve">
-    <value>Open new window</value>
+    <value>Open a new window instance</value>
```

Comment:
`instance` is a bit redundant. I'm not against adding 'a', but it's worth considering that this will reset existing translations, and it might be best to just keep the current string.